### PR TITLE
Removing quarterly bond reporting and updating ms report IDs

### PR DESCRIPTION
**Pull request by Charlie-Henry**
https://github.com/cityofaustin/atd-data-tech/issues/18740

[related airflow PR](https://github.com/cityofaustin/atd-airflow/pull/250)

This PR cleans up a lot of unused code in this repo.
- Removes all of the quarterly reporting stuff we don't use anymore.
- I copied the microstrategy reports referenced to a DTS folder (they were in Meredith's personal my reports previously), so this changed the report IDs.
- Updated requirements.txt due to compatability issues surrounding the dang mstrio-py package

***

#### Docker
Build your env_file using the template. The [airflow dag ](https://github.com/cityofaustin/atd-airflow/blob/production/dags/atd_bond_reporting.py)shows where to get each of the secrets in 1pass.

be on VPN then cd to this repo and:
```
$ docker build -t atddocker/atd-bond-reporting .
$ docker run -it --env-file env_file atddocker/atd-bond-reporting /bin/bash
```

#### microstrategy_to_s3.py
You can test it with this report here:

```
$ python atd-bond-reporting/microstrategy_to_s3.py -r "2020 Bond Expenses Obligated"
```

#### bond_data.py and bond_calculations.py
This script loads csvs into postgres, and validates the schema:
```
$ python atd-bond-reporting/bond_data.py
```

This calculates running totals and other things then replaces socrata datasets:
```
$ python atd-bond-reporting/bond_calculations.py
```

All of these scripts should run without errors.

### CODE REVIEW

File: microstrategy_to_s3.py [hunk 1]
```diff
@@ -20,17 +19,16 @@
...
+    "2020 Bond Expenses Obligated": "D6BC5BD13143FF3129F3318F589EBD51",
+    "All bonds Expenses Obligated": "077C066E6C4BF56B081F96A3E405D83C",
 }
 # To find report ID, go to the report in Microstrategy then:
 ## Go to Tools > Report Details Page or Document Details Page.
 ## Click Show Advanced Details button at the bottom
+
+# All microstrategy reports are stored in the DTS Folder located at:
+# Financial Service Analytics/Shared Reports/Other Departments/Transportation Shared Reports/Data & Technology Services Reports
+# https://coa-prod.cloud.microstrategy.com:443/MicroStrategy/servlet/mstrWeb?evt=2001&src=mstrWeb.shared.fbb.fb.2001&folderID=E8CDB3C06B4EBCE74341739C543B2687&Server=ENV-327524LAIO1USE1&Project=Financial%20Services%20Analytics&Port=39321&share=1
```
Comment (johnclary) [inline microstrategy_to_s3.py:31]: 👍

**Review by johnclary — APPROVED** [review]
🟥 🟥 🟥 🟥 ✅

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM python:3.8-slim
+FROM python:3.10-slim
```
Comment (frankhereford) [inline Dockerfile:1]: Thanks for keeping us up to date 🙏🙏🙏

**Review by frankhereford — APPROVED** [review]
Looking good! Love all the red and the version updates. All the scripts ran without throwing an exception, but there are some warnings

`microstrategy_to_s3.py`
```
DeprecationWarning: mstrio.project_objects.dossier module is deprecated and will not be supported starting from mstrio-py 11.5.03. Please use mstrio.project_objects.dashboard instead.
```
`bond_calculations.py`
```
/app/atd-bond-reporting/bond_calculations.py:299: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df["date"] = pd.to_datetime(df["date"], infer_datetime_format=True)
/app/atd-bond-reporting/bond_calculations.py:300: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df["date"] = df["date"].dt.strftime(DATE_FORMAT_SOCRATA)
```

Not a big deal, but something to look for down the road. 🚢🚢🚢

**Review by chiaberry — APPROVED** [review]
🧽 thanks for the cleanup!

File: requirements.txt [hunk 3]
```diff
@@ -1,6 +1,7 @@
+mstrio-py==11.4.3.*
```
Comment (mddilley) [inline requirements.txt:1]: 👍

**Review by mddilley — APPROVED** [review]
All scripts ran without error! ✅ 🧹

**Pull request merged**